pkg/virt-config/featuregate/inactive.go

```diff
@@ -115,4 +124,6 @@ func init() {
 
 	RegisterFeatureGate(FeatureGate{Name: PasstGate, State: Discontinued, Message: PasstDiscontinueMessage, VmiSpecUsed: passtApiUsed})
 	RegisterFeatureGate(FeatureGate{Name: MacvtapGate, State: Discontinued, Message: MacvtapDiscontinueMessage, VmiSpecUsed: macvtapApiUsed})
+
+	RegisterFeatureGate(FeatureGate{Name: InstancetypeReferencePolicy, State: GA})
```

Comment:
@lyarwood do we still need this FG? At what point are we going to remove it?

Reply:
I thought we could remove graduated feature gates.

Reply:
We keep GA'd FGs around for now, mostly for in-tree documentation.

Reply:
Nit: Move it to the block of GAed FGs at L120?